src/tests/ChordStructure.test.js

```diff
@@ -22,6 +22,14 @@ test('seventh chord type returns five chord structures', () => {
   expect(chordStructures(ChordType.SEVENTH)).toEqual(expected)
 })
 
-test('ChordStructure.NEAPOLITAN_SIXTH exists', () => {
-  expect(ChordStructure.NEAPOLITAN_SIXTH).toBeDefined()
+test('Basic chord structures exist', () => {
```

Comment:
This may seem stupid for now, but functions as a nice security blanket. If someone screws something up by accident, we've got some code coverage here just in case.